misbehave.html

```diff
@@ -55,6 +55,17 @@
         behaviorDefinition[key] = thisPrototype[key];
       }
 
+      behaviorDefinition.injectFrom = function(base, target) {
+        var baseElement = document.createElement(base);
```

Comment:
The purpose of injectFrom is to be an accessory of .toBehavior and should only be used in a misbehaving element for the purpose of getting the inherited template. I think we should enforce that base be 'one of' the elements .toBehavior-ing in the ext component.